.github/workflows/jerigon-native.yml

```diff
@@ -58,41 +57,37 @@ jobs:
 
       #It is much easier to use cast tool in scripts so install foundry
       - name: Install Foundry
-        uses: foundry-rs/foundry-toolchain@v1          
+        uses: foundry-rs/foundry-toolchain@v1
 
       - name: Run cancun test network
         run: |
           docker pull ghcr.io/0xpolygonzero/erigon:feat-zero
```

Comment:
@Nashtare @atanmarko 
https://github.com/0xPolygonZero/erigon/pull/15
This PR merge is blocker for CI to run fine